src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,660 @@
+      "lineNumber": 11,
+      "message": "Document policy violation: document-write is not allowed in this document.",
+      "policyId": "document-write",
+      "sourceFile": "https://site.example/script.js"
+    },
+    "type": "document-policy-violation",
+    "url": "https://site.example/",
+    "user_agent": "Mozilla/5.0... Chrome/92.0.4504.0"
+  },
+  {
+    "age": 510,
+    "body": {
+      "blockedURL": "https://site.example/img.jpg",
+      "destination": "image",
+      "disposition": "enforce",
+      "type": "corp"
+    },
+    "type": "coep",
+    "url": "https://dummy.example/",
+    "user_agent": "Mozilla/5.0... Chrome/92.0.4504.0"
+  }
+]
+```
+
+{% endDetails %}
+
+{% Aside %} Note:
+
+- In the new Reporting API (v1), thee browser only batches together reports in the same
+  request if they are generated by the same document (page). Reports are not sent in the
+  same request if they come from different origins, or even different documents on the
+  same origin such as different pages on the same website. Learn more in the [migration
+  guide](/reporting-api-migration).
+- In the new Reporting API, reports are scoped to **documents**. This means that for one
+  given origin, different documents (`site.example/page1` and `site.example/page2`) can
+  send reports to different endpoints. If you're migrating from the legacy Reporting API
+  to the new Reporting API, check the [migration guide](/reporting-api-migration).
+- Reports are sent with a delay to increase the chances that reports are sent in batches.
+  This saves bandwidth to be respectful to the user's network connection. This is
+  especially important on mobile.
+
+{% endAside %}
+
+## Reports you get
+
+Let's assume that you've set up `Reporting-Endpoints` and a few policies on your page.
+
+✅ Reports that are generated due to violations of these policies (or deprecations) on your
+page will be sent to the endpoint(s) you've configured. This includes violations committed
+by **third-party scripts** running on your page.
+
+❌ Violations committed by a **cross-origin iframe** on your page will **not** be reported
+to your endpoint(s), to ensure privacy. The iframe could set up its own reporting and even
+report to your site's (i.e. the first-party's) reporting service, but that's a decision
+that the framed site would have to make.
+
+{% Aside 'gotchas' %}
+
+In Chrome DevTools, you'll see a console error or warning pop up for violations that are
+committed by third-party script but also by third-party iframes.
+
+{% endAside %}
+
+## Browser support
+
+The table below sums up browser support for the **Reporting API v1**, that is with the
+`Reporting-Endpoints` header. Browser support for the legacy Reporting API (`Report-To`
+header) is the same, except for one report type: Network Error Logging isn't supported in
+the new Reporting API. Read the [migration
+guide](/reporting-api-migration/#network-error-logging) for details.
+
+<div class="w-table-wrapper">
```

Comment:
I'll come back to this table formatting after I talk to Rob. Ideally, it would use only Yes/No in text and highlight the cells in green/red.

Reply:
SG! LMK when you have an update on this.
Side note: one thing I was looking at when preparing this table was [this example in the handbook](https://web.dev/handbook/web-dev-components/#:~:text=if%20you%20want%20content%20in) with the crosses and checks. IIUC it just happens to be a yes/no table, the reason it's in the handbook is to illustrate something else (alignment).

![image](https://user-images.githubusercontent.com/9762897/122388343-175a1b00-cf70-11eb-86d2-bd83d39f4471.png)

If yes/no tables are common, an example in the handbook on how to do these right would be super helpful!

Reply:
Thanks for bringing this to my attention, I'm gonna need to do a through review of use of tables on the site. 👷🏻‍♀️